manifests/Microsoft/VisualStudioCodeInsiders/1.46.0.yaml

```diff
@@ -0,0 +1,13 @@
+Id: Microsoft.VisualStudioCodeInsiders
+Version: 1.46.0
+Name: Visual Studio Code Insiders
+Publisher: Microsoft Corporation
+License: Copyright (c) Microsoft Corporation
+AppMoniker: vscode-insiders
+Description: Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications. Visual Studio Code is free and available on your favorite platform - Linux, macOS, and Windows.
+Homepage: https://code.visualstudio.com/insiders/
+Installers:
+  - Arch: x64
+    Url: https://go.microsoft.com/fwlink/?Linkid=852155
```

Comment:
This would probably be better: https://update.code.visualstudio.com/latest/win32-x64-user/insider, the hash won't work though as it changes every day. While certainly possible, it would be preferable not to need a PR (or automated commit) everyday to do this. I also saw that sources (`winget source`) are supported, are all Microsoft packages meant to be maintained in this repo? It would be easier to automate if we owned the repo.

Reply:
Scoop also had to deal with this in https://github.com/lukesampson/scoop/issues/181, and they [added simplistic nightly support](https://github.com/lukesampson/scoop/pull/314). Might be worth to see that they did.